packages/react/src/Form/Pass/PassInput.tsx

```diff
@@ -0,0 +1,97 @@
+import { ComponentProps, ReactNode, useId, useState } from "react";
+import { Field, FieldInput, HelpMessage, useInputClassModifier } from "../core";
+import { Pass } from "./Pass";
+
+const strengthList: Record<number, string> = {
+  0: "bad",
+  1: "okay",
+  2: "good",
+  3: "verygood",
+  4: "excellent",
+};
+
+const calculateStrength = (score?: string | null) => {
+  if (score === null || score === undefined) {
+    return null;
+  }
+  let strength = Number(score) ?? 0;
+  if (strength > 4) {
+    strength = 4;
+  }
+  if (strength < 0) {
+    strength = 0;
+  }
+  return strengthList[strength];
+};
+
+type PassProps = ComponentProps<typeof Pass>;
+type Props = ComponentProps<typeof Field> &
+  Omit<PassProps, "onToggleType" | "type"> & {
+    helpMessage?: ReactNode;
+    score?: string;
+  };
+
+const PassInput = ({
+  message,
+  children,
+  helpMessage,
+  id,
+  disabled = false,
+  label,
+  messageType,
+  isVisible,
+  forceDisplayMessage,
+  className,
+  classModifier,
+  classNameContainerLabel,
+  classNameContainerInput,
+  score,
+  ...passProps
+}: Props) => {
+  const strength = calculateStrength(score);
+  const classModifierStrength = [classModifier ?? "", strength ?? ""].join(" ");
+
+  const [type, setType] = useState<PassProps["type"]>("password");
+  const inputId = useId();
+  const finalId = id ?? inputId;
```

Comment:
```suggestion
  const finalId = id || inputId;
```
en cas de chaine vide l'id générer sera utilisé

Reply:
cf #79

Reply:
yes mais que fait on si l'utilisateur envois une chaine vide ?
Oki l'issue propose un warning pour prévenir mais ici on utilise pas le code de issue donc pourquoi ?

Reply:
on attend que qqun prenne l'issue pour changer le behaviour.
On en a discuté la semaine derniere, mais grosso modo, si qqun a mis `id=""` c'est probablement pour une raison.
C'est pour ca qu'on ne force pas un id dans ce cas.

Et le code de l'issue c'est un exemple, pas l'implem finale forcément